### PR TITLE
Repaths /obj/item/weapon to /obj/item

### PR DESCRIPTION
**Pull request by Cyberboss**
Crappy bloat typepath with literally 1 var and 1 override that should probably be on /item. Merges items.dmi and weapons.dmi

`git grep -l "obj/item/weapon/" | xargs sed -i -b 's/obj\/item\/weapon\//obj\/item\//g'`

### CODE REVIEW

**Comment by MrStonedOne** [comment]
THE MAD MAN

**Comment by MoreRobustThanYou** [comment]
>  Files changed 1,088

HOLY.... OH MY GOD

**Comment by Cyberboss** [comment]
I'M NUCLYAIR!

**Comment by Bawhoppen** [comment]
how do you have the patience to do stuff like this

**Comment by Cyberboss** [comment]
>patience

`git grep -l "obj/item/weapon/" | xargs sed -i -b 's/obj\/item\/weapon\//obj\/item\//g'`

**Comment by Bawhoppen** [comment]
Yea, but there's also all kinds of references on the wiki and other guides aswell as other random places


Unless of course, you didn't do that stuff

**Comment by lzimann** [comment]
\>updating documentation
who do you think we are?

**Comment by KorPhaeron** [comment]
Gonna be a nightmare set of conflicts for every open PR

**Comment by Cyberboss** [comment]
Plebians git out

**Comment by WJohn** [comment]
>25,000 line changes

>6 GORRILION PATH CHANGES FOR ALL MAPS

oh i'm scared alright

**Comment by lzimann** [comment]
Conflicts are super easy to fix, the only exceptions for that is when dealing with maps and icons.

**Comment by MoreRobustThanYou** [comment]
fun? you mean fukkin chaos.

**Comment by DoctoreMad** [comment]
**SHIT CODE IS A RIGHT NOT A PRIVILEGE**

**Comment by optimumtact** [comment]
The question is why bother? Is there any proven benefit other than code neatness?

**Comment by ShizCalev** [comment]
@optimumtact no

**Comment by KorPhaeron** [comment]
@tgstation/commit-access we should decide on this and get it over with sooner rather than later

I think it is probably more trouble than its worth especially in regards to downstreams

**Comment by lzimann** [comment]
Currently test-merged, if there's no issue this can be merged(if that's what is decided).

**Comment by Cyberboss** [comment]
None whatsoever it seems

File: code/modules/admin/create_object.dm [hunk 1]
```diff
@@ -12,7 +12,7 @@
 	var/static/list/create_object_forms = list(
 	/obj, /obj/structure, /obj/machinery, /obj/effect,
 	/obj/item, /obj/item/clothing, /obj/item/stack, /obj/item/device,
-	/obj/item/weapon, /obj/item/weapon/reagent_containers, /obj/item/weapon/gun)
+	/obj/item, /obj/item/reagent_containers, /obj/item/gun)
```
Comment (Jordie0608) [inline code/modules/admin/create_object.dm]: Issue detected

**Comment by WJohn** [comment]
I mean this *is* going to cause like a million pathing errors on every map ever.

**Comment by Qustinnus** [comment]
He used regex on all files to prevent that.

**Comment by Cyberboss** [comment]
Maps have already been converted

**Comment by banthebantz** [comment]
i'm scared

File: code/modules/admin/create_object.dm (hunk 1 above)
Comment (Cyberboss) [inline code/modules/admin/create_object.dm]: Inconsequential, but good catch!

**Comment by DoctoreMad** [comment]
Holy shit you actually merged it?

**Comment by MoreRobustThanYou** [comment]
*scream
*scream

**Comment by Bawhoppen** [comment]
I mean like, any problems with it are super easy to fix

**Pull request merged**